Remove links to ythlsgo.onrender.com

freearhey opened the pull request:
Closes https://github.com/iptv-org/iptv/issues/14544

Link to download all deleted links: https://freenote.biz/D1qmkPDBmX

freearhey committed:
Update playlists

Dum4G commented:
This removes our knowledge of potentially working sources. Maybe move them to a dedicated playlist that won't be used until we get a stable YT scrapper?

freearhey commented:
Here's a complete list of all the links removed: https://freenote.biz/D1qmkPDBmX

Dum4G committed:
Create yt

Intentionally unused till we find some proper scrapper solution for demanded workload

freearhey commented on streams/yt:
```diff
@@ -0,0 +1,1411 @@
+#EXTM3U
```
I don't think it's necessary to store links in a repository.

freearhey commented:
I posted a link to all the links in Discussions as well: https://github.com/orgs/iptv-org/discussions/1535. And as soon as a replacement is found, we will definitely add them back.

freearhey committed:
Delete yt
